docs/configuration.md

```diff
@@ -1233,6 +1246,8 @@ Apart from these, the following properties are also available, and may be useful
   <td>
     If true, Spark will attempt to use off-heap memory for certain operations. If off-heap memory 
     use is enabled, then <code>spark.memory.offHeap.size</code> must be positive.
+    <em>Note:</em> If off-heap memory is enabled, raise the non-heap memory size (e.g. increase
```

Comment:
OK, rather than say "raise", I think the idea here is to suggest that you _may need to raise_ these values. You may not need to.

Reply:
I added "you may need to" as a suggestion.